projects/balancer/index.js

```diff
@@ -17,15 +19,27 @@ async function tvl(timestamp, ethBlock) {
   }
   `;
   const results = await graphQLClient.request(query, {
-    block: ethBlock
+    block
   })
-  return toUSDTBalances(results.balancers[0].totalLiquidity);
+  return results.balancers[0].totalLiquidity;
+}
+
+async function ethereum(timestamp, ethBlock) {
+  const [v1,v2] = await Promise.all([getTVL("balancer", ethBlock), getTVL("balancer-v2", ethBlock)])
```

Comment:
I see, so that's the right way to compute the TVL, was a bit confused earlier when trying to figure it out

https://twitter.com/0xngmi/status/1409167935088828417